### ISSUE TITLE
Use static provisioning to bind an already created LUN

### ISSUE THREAD

**Issue by tzago** [SynologyOpenSource/synology-csi#90]
Opening this tkt with same topic as issue #88 as this is the same issue I am facing.
My case is a Synology NAS migration from Snynology02 to new Synology01 box.
I used the Hyper Backup to back up luns from one synology nas to another.
All luns were backuped successfully although I noted that the backupd luns ISCSI IQN target got the same name as the LUN which starts with k8s-csi-pvc-*
While I noticed that the correct Target IQN name must start with iqn.2000-01.com.synology:x.pvc-* 
where x in my case is my Synology Hostname.

I confirm that my cloned Lun on my new synoloyg works fine while mounting it from the command line on my destination host.
something like
`sudo iscsiadm --mode node --targetname iqn.2000-01.com.synology:Synology01.pvc-xxxxxx-yyyy-zzzz-wwww-ea0db6860e76 --portal 192.168.0.1:3260,0 --login
`
But whenever I try to create a PV and PVC and use it on a new POD I get
```
"source":"","storage.kubernetes.io/csiProvisionerIdentity":"1726942005718-8081-csi.san.synology.com"},"volume_id":"a3bb180f-600f-455e-bc69-96773597ef28"}
2024-09-21T20:19:24Z [ERROR] [driver/utils.go:108] GRPC error: rpc error: code = Internal desc = rpc error: code = NotFound desc = Volume[a3bb180f-600f-455e-bc69-96773597ef28] is not found
2024-09-21T20:21:26Z [INFO] [driver/utils.go:104] GRPC call: /csi.v1.Node/NodeGetCapabilities
```
I am assuming here that what I am really missing to understand is how we can figure out what is the volumeHandle that I must set in my PV to correctly correlate it to my Cloned LUN in my new Synology which I already proved it is working via iscsiadm via cli.

In short, what is the correlation of volumeHandle and the Synology ISCSI IQN target and where I can fidn what is the correct volumeHandle UID to use?

**Comment by james-callahan** [comment]
> where I can find what is the correct volumeHandle UID to use?

I ended up using `synoiscsiwebapi lun list` on my synology box, the `uuid` is what to use as the `volumeHandle`. I was able to match it using the `name` field.
I can't see how to get this info in the synology web ui.

**Comment by mmontes11** [comment]
[The solution](https://github.com/SynologyOpenSource/synology-csi/issues/90#issuecomment-2407345260) from @james-callahan worked for me!
